core/primitives/src/errors.rs

```diff
@@ -332,6 +332,48 @@ pub struct ActionError {
     pub kind: ActionErrorKind,
 }
 
+#[derive(
+    BorshSerialize, BorshDeserialize, Debug, Clone, PartialEq, Eq, Deserialize, Serialize, RpcError,
+)]
```

Comment:
Should we

```suggestion
#[derive(Debug, Clone, PartialEq, Eq)]
```
?

My understanding is that, ideally we indeed would prefer to use this serialization format. But as we are not using this serialization right now, we might avoid derives? Seems easy enough to add them when we need them.

Reply:
Exactly the case, I'd love that to be serialization format, but it's not.